.circleci/config.yml

```diff
@@ -384,7 +384,12 @@ install_additional_cluster: &install_additional_cluster
       } || {
         echo "Error while creating the additional cluster after retry"
       }
-
+load_kind_images: &load_kind_images
+  run:
+    name: "Load needed images into Kind"
+    command:
+      ./script/load-kind-image.sh docker.io/bitnami/apache:2.4.48-debian-10-r74 kubeapps-ci kubeapps-ci-additional &&
+      ./script/load-kind-image.sh docker.io/bitnami/apache:2.4.48-debian-10-r75 kubeapps-ci kubeapps-ci-additional
```

Comment:
Question: why is it required? I mean, shouldn't the cluster pull the images if necessary?

Reply:
It turns out that there were some runs where tests failed due to those images not being present and failing to be pulled upon pod creation. FYI, at some runs  `2.4.48-debian-10-r74` would be successfully pulled but `2.4.48-debian-10-r75` would fail with a `401`!. Since images are pre-pushed I haven't seen an error regarding this.